cftlib/nfdiv/src/main/java/uk/gov/hmcts/divorce/solicitor/event/SolicitorRoles.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.divorce.solicitor.event;
+
+import lombok.Getter;
+import uk.gov.hmcts.divorce.divorcecase.model.UserRole;
+
+@Getter
+public enum SolicitorRoles {
```

Comment:
can we move all this code inline into CreateTestCase?

Reply:
done